internal/strings/strings_test.go

```diff
@@ -89,3 +92,46 @@ func TestRandomStringConcurrency(t *testing.T) {
 		go RandomString(10000)
 	}
 }
+
+func TestAsciiToLower(t *testing.T) {
+	hw := "HELLO WORLD"
+	hw = AsciiToLower(hw)
+	if hw != "hello world" {
+		t.Errorf("AsciiToLower() = %s, want %s", hw, "hello world")
+	}
+}
+
+func BenchmarkAsciiVsUnicodeCaseString(b *testing.B) {
+	strs := []string{
+		"This is a String With a lot of Characters!!!",
+		"this is a lowercase string with many characters",
+		"THIS IS AN UPPERCASE STRING WITH MANY CHARACTERS",
+		"ThIs Is A StRiNg WiTh MiXeD CaSe",
+	}
+	b.Run("hacky ascii", func(b *testing.B) {
+		for i := 0; i < b.N; i++ {
+			for _, str := range strs {
+				AsciiToLower(str)
+			}
+		}
```

Comment:
Needs to separate the benchmarks for the cases

```suggestion
		for _, str := range strs {
			b.Run(str, func(b *testing.B) {
				for i := 0; i < b.N; i++ {
					AsciiToLower(str)
				}
			}
		}
```